### PR TITLE
notPseudonym Transformer update

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# notPseudonym field transformer update

Updated the metadata model and transformer according to resolved discussion on gitlab.

### What's changed?
* notPseudonym field moved from keyType to variable
* transformer update for new model
* updated all testdata

### CODE REVIEW

**Review by vaskalan — CHANGES_REQUESTED** [review]
Can't find any unit tests verifying that notPseudonym is transformed correctly.
Correct me if I'm wrong.

File: tests/resources/worker/steps/transformer/expected/KREFTREG_DS_enumerated.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
       "variableRole": "Identifier",
```
[the remark this answers is not in the document]
Comment (vaskalan) [reply tests/resources/worker/steps/transformer/expected/KREFTREG_DS_enumerated.json:17]: True, but where are the unit tests located?

[the remark this answers is not in the document]
Comment (DanielElisenberg) [reply tests/resources/worker/steps/transformer/expected/KREFTREG_DS_enumerated.json:17]: * [transformer tests](https://github.com/statisticsnorway/microdata-job-executor/blob/master/tests/unit/worker/steps/test_dataset_transformer.py)
* [build metadata test](https://github.com/statisticsnorway/microdata-job-executor/blob/master/tests/unit/worker/test_build_metadata_worker.py)

[the remark this answers is not in the document]
Comment (vaskalan) [reply tests/resources/worker/steps/transformer/expected/KREFTREG_DS_enumerated.json:17]: Thank you, and sorry for the delay 😃

**Review by vaskalan — APPROVED** [review]

**Pull request merged**